randovania/games/samus_returns/exporter/game_exporter.py

```diff
@@ -25,10 +51,46 @@ def export_can_be_aborted(self) -> bool:
         """
         return False
 
+    def _before_export(self) -> None:
+        assert not self._busy
+        self._busy = True
+
+    def _after_export(self) -> None:
+        self._busy = False
+
     def _do_export_game(
         self,
         patch_data: dict,
         export_params: GameExportParams,
         progress_update: status_update_lib.ProgressUpdateCallable,
-    ):
-        raise NotImplementedError
+    ) -> None:
+        assert isinstance(export_params, MSRGameExportParams)
+        export_params.output_path.mkdir(parents=True, exist_ok=True)
```

Comment:
I guess you can't test this function if the import is not included? 🤔 
Maybe keep just the import lines commented out or in a `if False:` block? or maybe even a try/except

Reply:
Added a weird test. Will be changed anyways in the next 1-2 weeks.